build-go.sh

```diff
@@ -303,9 +303,9 @@ function currentSha() {
 }
 
 function printVersions() {
-	local versfile="$1"
-	versarr=$(tr "\n" ' ' < "$versfile")
-	echo "building versions: $versarr"
+	local versions="$1"
```

Comment:
The list of versions that need building is now calculated during the build, so we change this to expect a list rather than a file name.